cmd/config_options.go

```diff
@@ -15,51 +15,123 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
 	"fmt"
-	"log/slog"
+	"github.com/falcosecurity/falcoctl/pkg/options"
+	"github.com/falcosecurity/falcoctl/pkg/output"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"io"
 	"os"
+	"strings"
 
 	"github.com/creasty/defaults"
 	"github.com/falcosecurity/driverkit/validate"
 	"github.com/go-playground/validator/v10"
+	"github.com/pterm/pterm"
 )
 
 var validProcessors = []string{"docker", "kubernetes", "kubernetes-in-cluster", "local"}
 var aliasProcessors = []string{"docker", "k8s", "k8s-ic"}
-var configOptions *ConfigOptions
 
 // ConfigOptions represent the persistent configuration flags of driverkit.
 type ConfigOptions struct {
 	ConfigFile string
-	LogLevel   string `validate:"loglevel" name:"log level" default:"INFO"`
 	Timeout    int    `validate:"number,min=30" default:"120" name:"timeout"`
 	ProxyURL   string `validate:"omitempty,proxy" name:"proxy url"`
 	DryRun     bool
 
-	configErrors bool
+	// Printer used by all commands to output messages.
+	Printer *output.Printer
```

Comment:
TODO: double check if `Printer` and `Writer` can be package private.

Reply:
These changes all make sense to me, but commenting here just to make sure you've resolved this TODO (if you wanted to in this MR)

Reply:
Yes! I could not make the change :)